lib/ansible/plugins/lookup/fileglob.py

```diff
@@ -16,6 +16,10 @@
       _terms:
         description: path(s) of files to read
         required: True
+      ignore_missing:
+        description: Flag to control whether or not no matches found return a warning
+        type: boolean
+        default: False
```

Comment:
version_added is missing

Reply:
thanks, fixed it. Also added a changelog entry.